### DISCUSSION TITLE
Shortcode usage on index.php

### DISCUSSION THREAD

**Discussion by ersingo** [bootscore/bootscore#371] [Q&A] [answered]
Hello Basti;

Can I place these codes not from the editor, but in the children's theme indexphp? i want to make a default homepage.

**Comment by crftwrk** [comment, marked as answer]
Use the `do_shortcode` to wrap the shortcode like this:

```php
<?php echo do_shortcode('[bs-swiper-card type="post" category="cars, boats" order="ASC" orderby="date" posts="6"]'); ?>
```

Now you can use shortcode in `.php` files.

**Comment by ersingo** [comment]
thank you very much. sorry about my inexperience. i didn't know this could happen.  @crftwrk

**Reply by crftwrk** [reply]
> sorry about my inexperience.

Using shortcodes in php files is a serious question and other users will also find the answer very helpful.

**Comment by ersingo** [comment]
Basti, hello again. I don't know if I should bring up another topic. Is there a code where I can show only the post titles with this plugin? It will list the post titles.

* Article title 1
* Article title 2
* Article title 3
* Article title 4
* Article title 5

**Reply by crftwrk** [reply]
Of course, it's simple. First override the plugin's template you are using in your child as described here https://bootscore.me/documentation/bs-swiper/#Overriding_templates_via_theme.

Open that file and inside you will find the snippets that shows image, title, excerpt etc. For example the card template https://github.com/bootscore/bs-swiper/blob/main/templates/sc-swiper-card.php

#### Featured image
```php
<?php the_post_thumbnail('medium', array('class' => 'card-img-top')); ?>
``` 

#### Category badge
```php
<?php bootscore_category_badge(); ?>
``` 

#### Title
```php
<h2 class="blog-post-title">
  <a href="<?php the_permalink(); ?>">
    <?php the_title(); ?>
  </a>
</h2>
```

#### Meta 
```php
<?php if ('post' === get_post_type()) : ?>
  <small class="text-muted mb-2">
    <?php
    bootscore_date();
    bootscore_author();
    bootscore_comments();
    bootscore_edit();
    ?>
  </small>
<?php endif; ?>
```

#### Excerpt
```php
<?php the_excerpt(); ?>
```

#### Read more
```php
<a class="read-more" href="<?php the_permalink(); ?>"><?php _e('Read more »', 'bootscore'); ?></a>
```

#### Tags
```php
<?php bootscore_tags(); ?>
``` 

Delete all snippets and keep only the title.
